### DISCUSSION TITLE
Serenity Test cases are not working in chrome 116 browser version but its working in chrome 104 as browser is launching for a second and it exits

### DISCUSSION THREAD

**Discussion by syedchay** [serenity-bdd/serenity-core#3242] [Q&A]
Hi All,

I have used the below build .gradle file and the test cases were working fine in chrome 104 but not working in 116 chrome browser version update. Kindly help on what needs to be modified.

build.gradle file:

defaultTasks 'clean','test','aggregate'
apply plugin: 'java'
apply plugin: 'net.serenity-bdd.aggregator'

group 'org.testing.syed'
version '1.0-SNAPSHOT'

repositories {
    mavenCentral()
    }
buildscript {
    project.ext {
        serenityVersion='2.3.12'
    }
    repositories {
        mavenCentral()
    }
    dependencies {
        classpath('net.serenity-bdd:serenity-gradle-plugin:'+ serenityVersion)
    }
}
sourceCompatibility=1.8
targetCompatibility=1.8
ext{
    slf4jVersion = '1.7.7'
    serenityCoreVersion = '2.3.12'
    junitVersion = '4.13'
    assertJVersion = '3.8.0'
    logbackVersion = '1.2.3'
}

dependencies {

    testImplementation "net.serenity-bdd:serenity-core:${serenityCoreVersion}",
            "net.serenity-bdd:serenity-junit:${serenityCoreVersion}",
            "net.serenity-bdd:serenity-screenplay:${serenityCoreVersion}",
            "junit:junit:${junitVersion}",
            "org.assertj:assertj-core:${assertJVersion}"
}

test{
    testLogging.showStandardStreams=true
    systemProperties System.getProperties()
}
gradle.startParameter.continueOnFailure=true
test.finalizedBy(aggregate)

**Comment by wakaleo** [comment]
You will need to upgrade to Serenity 4.0.0

**Reply by syedchay** [reply]
Hi John,

Its still not working with just updating the serenity core version. 
Its throwing the below error while building gradle project(I did only two changes-net.serenity-bdd:serenity-gradle-plugin: 2.4.4 and net.serenity-bdd:serenity-core:4.0.0-beta-1 
Could not resolve com.google.guava:guava:31.1-jre.

Can you please let me know if anything additional needs to be updated in build.gradle. 

PFB git repo for the same. 
[****](https://github.com/syedchay/TestingProject.git)

Thanks and Regards,
Atif
